AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -209,7 +209,7 @@ playngo.com##.deobxn
 bbc.com#?#header[role="banner"] > div > div:has(> div  > div > h2)
 ||sportrar.tv/js/cookiepop.js
 dirt.com,deadline.com,variety.com,billboard.com,indiewire.com###pmc_toupp_notice
-||pmc-pp-tou.*.amazonaws.com/privacy-notice.js$domain=indiewire.com
+||pmc-pp-tou.*.amazonaws.com/privacy-notice.js$domain=hollywoodreporter.com|indiewire.com
```

Comment:
I changed the order of the domains a bit. It's kinda convient to add them from the left, especially when you'll have a lot of domains.
Btw, the rule work fine.